src/System.Net.Security/tests/Scripts/setup-kdc.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
```

Comment:
@vijaykota I wouldn't expect this to succeed as a part of automated runs.  The first call to sudo is likely going to password prompt.  If the process's stdin makes it to the keyboard that'll hang.  If it's a static channel it'll fail.  Neither of those results in success.

I think that the tests will just need to use [ConditionalFact], not run on most people's machines, and we'll probably want to come up with a way of having a test fail if the condition is false and it's an official machine.